js/index.js

```diff
@@ -38,5 +38,106 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Image of a code snippet
+
+let img = document.getElementById("cta-img");
+img.setAttribute('src', siteContent["cta"]["img-src"])
+
+// Navigation
+
+let navA = document.querySelectorAll('a')
+
+navA[0].setAttribute('test', '.nav-item-1')
+navA[0].text = 'Services'
+navA[1].setAttribute('test', '.nav-item-2')
+navA[1].text = 'Product'
+navA[2].setAttribute('test', '.nav-item-3')
+navA[2].text = 'Vision'
+navA[3].setAttribute('test', '.nav-item-4')
+navA[3].text = 'Features'
+navA[4].setAttribute('test', '.nav-item-5')
+navA[4].text = 'About'
+navA[5].setAttribute('test', '.nav-item-6')
+navA[5].text = 'Contact'
+
```

Comment:
Here is an example of doing it the way we talked about in stand up using the forEach method. This code loops through the nodelist you have selected and assigns values from the siteContent object.

```
nav.forEach(function(value, i){
  value.textContent=siteContent.nav[`nav-item-${i+1}`];
});
```